keyboards/handwired/myskeeb/keymaps/default/default_e6b9980.json

```diff
@@ -0,0 +1 @@
+{"keyboard":"ergodash/rev1","keymap":"default_e6b9980","layout":"LAYOUT","layers":[["KC_ESC","KC_1","KC_2","KC_3","KC_4","KC_5","KC_GRV","KC_MINS","KC_6","KC_7","KC_8","KC_9","KC_0","KC_BSPC","KC_CAPS","KC_Q","KC_W","KC_E","KC_R","KC_T","KC_LBRC","KC_RBRC","KC_Y","KC_U","KC_I","KC_O","KC_P","KC_EQL","KC_TAB","KC_A","KC_S","KC_D","KC_F","KC_G","KC_NO","KC_NO","KC_H","KC_J","KC_K","KC_L","KC_SCLN","KC_QUOT","KC_LSFT","KC_Z","KC_X","KC_C","KC_V","KC_B","KC_CALC","KC_PSCR","KC_N","KC_M","KC_COMM","KC_DOT","KC_SLSH","KC_RSFT","KC_LCTL","KC_LGUI","KC_LALT","KC_DEL","LT(1,KC_BSPC)","KC_SPC","KC_ENT","KC_ENT","KC_SPC","KC_APP","KC_LEFT","KC_RGHT","KC_UP","KC_DOWN"],["KC_ESC","KC_F1","KC_F2","KC_F3","KC_F4","KC_F5","KC_F6","KC_F7","KC_F8","KC_F9","KC_F10","KC_F11","KC_F12","KC_BSPC","KC_CAPS","KC_PDOT","KC_PPLS","KC_PMNS","KC_PAST","KC_PSLS","KC_MSEL","KC_MSTP","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_PWR","RESET","KC_TAB","KC_P6","KC_P7","KC_P8","KC_P9","KC_P0","KC_NO","KC_NO","KC_SLCK","MAGIC_HOST_NKRO","EEP_RST","KC_TRNS","KC_TRNS","KC_TRNS","KC_LSFT","KC_P1","KC_P2","KC_P3","KC_P4","KC_P5","KC_MUTE","KC_MPLY","KC_NLCK","MAGIC_UNHOST_NKRO","KC_TRNS","KC_TRNS","KC_TRNS","KC_TRNS","KC_LCTL","KC_LGUI","KC_LALT","KC_BSPC","KC_TRNS","KC_VOLD","KC_VOLU","KC_MPRV","KC_MNXT","TO(2)","KC_HOME","KC_END","KC_PGUP","KC_PGDN"],["KC_ESC","KC_1","KC_2","KC_3","KC_4","KC_5","KC_6","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_TAB","KC_Q","KC_W","KC_E","KC_R","KC_T","KC_7","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_LCTL","KC_A","KC_S","KC_D","KC_F","KC_G","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_LSFT","KC_Z","KC_X","KC_C","KC_V","KC_B","KC_SPC","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_SPC","KC_SPC","KC_SPC","KC_NO","KC_NO","TO(3)","KC_NO","KC_NO","KC_LALT","KC_TAB"],["KC_ESC","KC_1","KC_2","KC_3","KC_4","KC_5","KC_6","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_TAB","KC_A","KC_W","KC_E","KC_R","KC_T","KC_H","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_LSFT","KC_Z","KC_S","KC_D","KC_F","KC_G","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_LCTL","KC_NO","KC_X","KC_C","KC_V","KC_B","KC_M","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_NO","KC_SPC","KC_SPC","KC_Q","KC_NO","KC_NO","TO(0)","KC_NO","KC_NO","KC_LALT","KC_TAB"]],"author":"","notes":""}
```

Comment:
Why is there an Ergodash keymap here?

Reply:
Hi. i downloaded the ergodash json file from qmk configurator. i used it as a base. can be deleted

Reply:
You can remove this file from the pull request by entering

```
git rm --cached -- keyboards/handwired/myskeeb/keymaps/default/default_e6b9980.json
```

in the command line. The `--cached` argument means the file will remain on your computer; leave it out if you want to delete the file from your system as well.